modules/popup/react/lib/Popup.tsx

```diff
@@ -152,7 +152,6 @@ export default class Popup extends React.Component<PopupProps> {
               size={closeIconSize}
               onClick={handleClose}
               icon={xIcon}
-              title={closeLabel}
```

Comment:
We already have an aria label which is why removed the title, thoughts?

Reply:
I think it was added so that a mouse user could see what the Icon would do? I agree we don't want both `title` and `aria-label`. Perhaps this should be a label Tooltip? The new Tooltip API only exists on the v4 branch, but would be what we want.

It would look something like:
```tsx
<Tooltip title={closeLabel}>
  <IconButton
    /* ... */
  />

The label is actually required on the tooltip as well as the `IconButton` because of Typescript. The Tooltip uses `React.cloneElement` and adds an `aria-label` to the child it wraps.
</Tooltip>